pandas/tests/dtypes/cast/test_promote.py

```diff
@@ -654,15 +591,16 @@ def test_maybe_promote_any_with_datetime64(
     )
 
 
-# override parametrization due to to many xfails; see GH 23982 / 25425
-@pytest.mark.parametrize("box", [(True, object)])
```

Comment:
@jbrockmendel 
here we something fell through the cracks in #23982 - this tests never ran with `box=False`.